pbjs-genfiles/README.md

```diff
@@ -0,0 +1,5 @@
+This folder is used for auto-generated JavaScript files and TypeScript type definitions
+generated by `pbjs` (from `protobufjs`) based on the protobuf
```

Comment:
could you link `pbjs` and `protobufjs` to their respective repos, so that folks can read up on the projects.

Reply:
This folder does not exist anymore!